### DISCUSSION TITLE
Tree Table - Multiple Selection using Shift Key

### DISCUSSION THREAD

**Discussion by aciesla4** [primefaces/community#1135] [PrimeNG] [answered]
Is there a way to use the shift key for multiple selection in a tree table? 

Currently, if I click on a node, hold the Shift key, and click on another node, only the second node is selected. 

I would like to click on a node, hold the Shift key, click on another node, and have all the nodes between the first and second select. 

Angular Version: 17.0.9
PrimeNG Version: 17.3.2

**Comment by cetincakiroglu** [comment]
Hi,

It should be possible by enabling `multiple`. Could you please try it https://primeng.org/treeselect#multiple

**Reply by aciesla4** [reply]
This does not work. In the example linked, if I click Meeting, hold the Shift key, then click Report Review, only Meeting and Report Review are selected. I would like Meeting, Product Launch, and Report Review selected. 

![image](https://github.com/primefaces/community/assets/43011130/7d3a3666-d7e9-4ee2-a237-290bde9bf3ee)

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

You're trying to select range and range selection is not supported by the Tree component yet. You can select multiple but you cannot select range for now. However it might be a good feature and we'll wait for community upvotes to implement it.
